.travis.yml

```diff
@@ -5,13 +5,13 @@ python:
   - "2.7"
 
 install:
-    - pip install pep8
+    - pip install pep8 tox
 
 before_script:
-    - "pep8 ./zask ./tests"
+    - "pep8 --ignore=E123,E402,E501,E701 ./bin ./zask ./tests"
```

Comment:
When I run `pep8 ./bin ./zask ./tests` I only see E402 errors. Is it not enough to ignore that one?